scrapi/migrations.py

```diff
@@ -19,97 +14,87 @@
 @tasks.task_autoretry(default_retry_delay=30, max_retries=5)
 def rename(docs, target=None, **kwargs):
     assert target, "To run this migration you need a target."
+
     for doc in docs:
-        raw = RawDocument({
-            'doc': doc.doc,
-            'docID': doc.docID,
-            'source': target,
-            'filetype': doc.filetype,
-            'timestamps': doc.timestamps,
-            'versions': doc.versions
-        })
+        raw_doc = doc.raw
+        new_doc = copy.copy(raw_doc.attributes)
+        new_doc['source'] = target
 
-        assert doc.source != target, "Can't rename {} to {}, names are the same.".format(doc.source, target)
+        raw = RawDocument(new_doc, validate=False)
```

Comment:
May want to have the option to run migrations with validate/clean, could make it another flag. Can be in a separate PR though, let's just get this merged for now.